package.json

```diff
@@ -13,7 +13,7 @@
     "history": "3.3.0",
     "jquery": "3.4.1",
     "patternfly": "3.59.1",
-    "patternfly-react": "1.19.1",
+    "patternfly-react": "2.34.3",
```

Comment:
It's a bit strange to update this dependency in this commit - it should be split out.

Reply:
Oops, this snuck in from a `npm audit fix`. I removed it completely from the PR.